l1-contracts/terraform/main.tf

```diff
@@ -6,18 +6,72 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "3.74.2"
+      version = "5.29.0"
     }
   }
 }
 
+data "terraform_remote_state" "aztec2_iac" {
+  backend = "s3"
+  config = {
+    bucket = "aztec-terraform"
+    key    = "aztec2/iac"
+    region = "eu-west-2"
+  }
+}
+
 variable "DEPLOY_TAG" {
   type = string
 }
 
 # S3 Bucket to store contract addresses
 resource "aws_s3_bucket" "contract_addresses" {
-  bucket = "aztec-${var.DEPLOY_TAG}-deployments"
+  bucket = "static.aztec.network"
+}
+
+resource "aws_s3_bucket_website_configuration" "addresses_website_bucket" {
```

Comment:
Minor thing: I'd move the S3 bucket and & cloudfront distribution to a separate terraform file since they're not strictly related to L1 contracts. I think it will make discovering the source code easier?

Reply:
I agree. This stuff should be elsewhere

Reply:
Agreed, moving to `iac` folder